### ISSUE TITLE
Uncaught Error: Call to undefined method WP2D_Options::attempt_password_upgrade()

### ISSUE THREAD

**Issue by bekopharm** [DiasPHPora/wp-to-diaspora#170]
Upgrading to 3.0.0 breaks access to `/wp-admin/` resulting in Fatal Error.

PHP is 7.3
WordPress is 5.4.2

This is the php error log:
```
2020/06/17 11:31:49 [error] 2601#2601: *1124613 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined method WP2D_Options::attempt_password_upgrade() in htdocs/wp-content/plugins/wp-to-diaspora/lib/class-wp2d.php:210
Stack trace:
#0 htdocs/wp-includes/class-wp-hook.php(287): WP2D->upgrade()
#1 htdocs/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#2 htdocs/wp-includes/plugin.php(478): WP_Hook->do_action()
#3 htdocs/wp-admin/admin.php(170): do_action()
#4 htdocs/wp-admin/index.php(10): require_once('...')
#5 {main}
```

Moved `wp-to-diaspora` from `htdocs/wp-content/plugins` to somewhere else temporary to access `/wp-admin/` again.

**Comment by noplanman** [comment]
@bekopharm so sorry about that!

Clearly the deploy script was missing something, should be ok now with 3.0.1.

🙏

**Comment by bekopharm** [comment]
Can confirm this fixed with 3.0.1, ty :+1:
